gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -150,6 +151,7 @@ enum
 #define TRIGGER "trigger"
 #define CHANNELS "scan_elements"
 #define IIO "iio:"
+#define TIMESTAMP "timestamp"
```

Comment:
This is a trivial comment. Some string constants have names like ${NAME}_PREFIX  and others have names like just ${NAME}.  Is it intended something with those names?

Reply:
${NAME} gives name for the element - this element can be device, trigger, buffer, etc
${NAME}_PREFIX is gives prefix for their file/directory name when filtering for the corresponding files/directory.

Please suggest better naming and I will update :)

Reply:
I really have no idea about what are good variable names. I just asked:)

Reply:
:)